.envrc

```diff
@@ -1,4 +1,6 @@
+HOST_XDG_DATA_DIRS="${XDG_DATA_DIRS:-}"
```

Comment:
I'd really like to get rid of lorri :( I think it's overengineered and with flakes not really needed.